packages/drivers/odsp-driver/src/odspDriverUrlResolverForShareLink.ts

```diff
@@ -139,30 +138,26 @@ export class OdspDriverUrlResolverForShareLink implements IUrlResolver {
             throw new Error("Failed to get share link because necessary information is missing " +
                 "(e.g. siteUrl, driveId or itemId)");
         }
+
         const key = this.getKey(resolvedUrl);
         const cachedLinkPromise = this.sharingLinkCache.get(key);
         if (cachedLinkPromise) {
             return cachedLinkPromise;
         }
-        const newLinkPromise = getShareLink(
-            this.getSharingLinkToken,
+        const newLinkPromise = getFileLink(
+            this.instrumentedTokenFetcher,
             resolvedUrl.siteUrl,
             resolvedUrl.driveId,
             resolvedUrl.itemId,
             this.identityType,
             this.logger,
-            "existingAccess",
-            undefined,
-            this.msGraphOrigin,
-        ).then((shareLink) => {
-                if (!shareLink) {
-                    throw new Error("Failed to get share link");
-                }
-                return shareLink;
-        }).catch((error) => {
-            if (this.logger) {
-                this.logger.sendErrorEvent({ eventName: "FluidFileUrlError" }, error);
+        ).then((fileLink) => {
+            if (!fileLink) {
+                throw new Error("Failed to get share link");
```

Comment:
nit: "share" -> "file"

Reply:
Keeping share since we refer to it as share link throughout this file